sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -111,13 +111,11 @@ trait SchemaRelationProvider {
 }
 
 /**
- * ::Experimental::
  * Implemented by objects that can produce a streaming `Source` for a specific format or system.
  *
  * @since 2.0.0
  */
-@Experimental
-@Unstable
+@Stable
```

Comment:
BTW, unstable to stable sounds a big jump tho. Maybe `Evolving` or simply removing `Unstable` can be a okay compromise if we're not sure on this.

Reply:
cc @tdas and @zsxwing

Reply:
@HyukjinKwon I don't think it makes sense to go through the motions step by step just for the sake of taking the steps. This interface has gone unchanged since Spark 2.2.0.